school_event/models/event.py

```diff
@@ -135,6 +135,15 @@ def _search(self, args, offset=0, limit=None, order=None, count=False,
                                access_rights_uid=access_rights_uid)
 
     @api.multi
+    def event_open(self):
+        for rec in self:
+            if rec.part_ids and len(rec.part_ids) >= 1:
```

Comment:
Hello sir , here we check how many participants are in the event. If more than 1 participant then event will start otherwise it will raise warning message. So we have used len().